apps/crn-frontend/src/network/teams/__tests__/TeamOutput.test.tsx

```diff
@@ -39,13 +39,13 @@ const mandatoryFields = async (
   {
     link = 'http://example.com',
     title = 'example title',
-    description = 'example description',
```

Comment:
What is the plan for migration? It was my understanding that we would maintain the old field but try and convert it into MD in the FE upon an old RO edit. This seems only to give the new MD field in which case I am not sure why the name changed.

Reply:
That's correct but to be addressed in a separate ticket. At the moment both field coexist, the old one is not mandatory.

Reply:
Why is the new one mandatory? I think some of the document types don't require a description

Reply:
if you look in `apps/crn-server/src/validation/research-output.validation.ts` you'll see the old one was mandatory